baseline/OSPS-QA.yaml

```diff
@@ -62,31 +62,21 @@ criteria:
     security_insights_value: # TODO
 
   - id: OSPS-QA-03
-    maturity_level: 2
+    maturity_level: 1
     criterion: |
-      All released software assets MUST be
-      delivered with a machine-readable list of
-      all direct and transitive internal software
-      dependencies with their associated version
-      identifiers.
+      The code repository should contain a dependency 
+      list that accounts for the direct language dependencies
+      when the package management system supports it.
```

Comment:
We haven't written anything tracking this requirement yet, but rewriting these criteria between RCs will be more of a challenge to coordinate in the future as we start measuring pilot projects against them.

(Fine for now)